components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -104,10 +120,15 @@ public function executeCommand(): void
         }
     }
 
-    private function forwardToPageObject(): void
+    private function forwardToPageObject(AuthPageEditorContext $context): void
```

Comment:
Do we really need to introduce a new parameter if we already have the context stored in `$this->context`?